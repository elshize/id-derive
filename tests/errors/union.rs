use id_derive::Id;

#[derive(Id)]
union Test {
    f1: u32,
    f2: u32,
}

fn main() {}
