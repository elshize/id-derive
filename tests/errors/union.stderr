error: Cannot derive Id for union, expected struct.
 --> $DIR/union.rs:4:1
  |
4 | union Test {
  | ^^^^^
